docs/classic-vs-vt.md

```diff
@@ -51,14 +51,16 @@ As Windows has evolved, the security controls and restrictions on window handles
 
 ### Unicode
 
-The implicit standard for communication across platforms and the web is Unicode, specifically in the UTF-8 form. Of course, other encodings still exist.  However, when otherwise undefined, using UTF-8 is widely accepted as the appropriate default. It represents the balance of portability, storage/transmission size, and breadth of expression required to support the world's languages and glyphs.
+The implicit standard for communication across platforms and the web is Unicode, specifically in the UTF-8 form. Of course, other encodings still exist. However, when otherwise undefined, using UTF-8 is widely accepted as the appropriate default. It represents the balance of portability, storage/transmission size, and breadth of expression required to support the world's languages and glyphs. However, an algorithmic equivalent in UTF-16 was chosen as the primary Unicode support mechanism for the Windows platform.

```

Comment:
Whole sentence seems needlessly wordy:

```
UTF-8 is the accepted encoding for Unicode data across almost all modern platforms, as it strikes the right balance between portability, storage size and processing time. This does not preclude the use of other encodings, of course: Windows uses UTF-16.
```

Reply:
maybe work from there? There's too many "specifically" and "form" and "However" and "support mechanism" in the original.

Reply:
k I made a revision we cleared offline.